src/DearImGui.hs

```diff
@@ -1088,19 +1093,60 @@ vSliderScalar label size dataType ref refMin refMax format flags = liftIO do
 
 -- | Wraps @ImGui::InputText()@.
 inputText :: (MonadIO m, HasSetter ref String, HasGetter ref String) => String -> ref -> Int -> m Bool
-inputText desc ref refSize = liftIO do
+inputText label ref bufBytes = liftIO do
   input <- get ref
-  withCString input \ refPtr -> do
-    withCString desc \ descPtr -> do
-      let refSize' :: CInt
-          refSize' = fromIntegral refSize
-      changed <- Raw.inputText descPtr refPtr refSize'
+  Foreign.withCString utf8 input \refPtr ->
+    Foreign.withCString utf8 label \labelPtr -> do
+      changed <- Raw.inputText
+        labelPtr
+        (refPtr, bufBytes)
```

Comment:
Using `refPtr` for a smaller string than the stated buffer size should not work. But I couldn't make it crash or something.

I tried to use `allocaBytes` instead to fit ref string and a possibly larger input value but it went bonkers, spamming changes and occasionally reading trailing garbage.